tests/ansible/tests/test_deploy.yaml

```diff
@@ -14,4 +14,4 @@
   vars:
     ssh_port: 12345
     output_dir: /tmp/test_ansible
-    docker_image: superbench/superbench
+    docker_image: superbench/superbench:v0.3.0-cuda11.1.1
```

Comment:
Do we need to change this?

Reply:
to avoid frequent image update and no space issue

Reply:
add comment, the image requires `sb` installed

Reply:
Got it, thanks!